platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -86,15 +83,9 @@ private HandlerResult checkAgainstCrl(HttpServletResponse httpResponse,
         } else {
```

Comment:
We're not setting the status code to 401 anymore, is that what we want? If so, we need to update the comment.